src/api/public/apidocs-new/paths/source_project_name_meta.yaml

```diff
@@ -33,3 +33,50 @@ get:
       $ref: '../components/responses/unknown_project.yaml'
   tags:
     - Sources - Projects
+
+put:
+  summary: Write project meta file
+  description: |
+    Write project's meta file. Create the project if it doesn't exist.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: query
+      name: comment
+      required: false
+      schema:
+        type: string
+      description: Comment that explains the changes you made in meta file.
+  requestBody:
+    description: Project meta definition.
+    content:
+      application/xml; charset=utf-8:
+        schema:
+          $ref: '../components/schemas/source/project_meta.yaml'
+        example:
+          name: home:Admin
+          title: Endless Night
+          description: Test project
+          person:
+            userid: Admin
+            role: maintainer
+          repository:
+            name: openSUSE_Tumbleweed
+            path:
+              project: openSUSE.org:openSUSE:Factory
+              repository: snapshot
+            arch:
+              - x86_64
+              - aarch64
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '400':
+      $ref: '../components/responses/bad_request_error.yaml'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      $ref: '../components/responses/unknown_project.yaml'
```

Comment:
```suggestion
```
I couldn't reproduce this response.

If the path parameter `project_name` is different to the `name` element passed in the `<project>` element of the body xml, I receive a 400 error:
```xml
<status code="project_name_mismatch">
  <summary>project name in xml data does not match resource path component</summary>
</status>
```

If they are both the same and the project doesn't exist, the project gets created. It's already in the description.